docs/best-practices/continuous-integration.md

```diff
@@ -446,3 +446,57 @@ The extension can be configured to install the latest version of _fastlane_ or a
 
 ## More Information
 For more documentation, see the [Apple App Store](https://marketplace.visualstudio.com/items?itemName=ms-vsclient.app-store) in the Visual Studio Team Server marketplace.
+
+# Nevercode Integration for iOS
+
+[Nevercode](www.nevercode.io) expects nothing more than a [`Fastfile`](https://docs.fastlane.tools/#fastlane) in your repository. 
+
+## Specify app repository
+
+1. Click **Add a new app** on Nevercode dashboard.
+
+2. Choose whether to add your application via supported providers using OAuth or add it manually using the other available [authentication methods](https://developer.nevercode.io/docs/authenticating-repository-access).
+
+## Define the lane to be built
+
+After specifying app repository, Nevercode takes a first look at it by listing the `branches` in this repository. The next step for you is to select the branch you want Nevercode to scan for projects.
+
+1. Select a `branch` from the dropdown that Nevercode should scan. **Note** that you can change the branch later if needed.
+2. Click **Scan branch**. 
+Nevercode scans the repository from the specified branch. This includes cloning the repository, looking up the available `Fastfiles` and listing the pre-configured `lanes`. All this can be monitored real-time from your browser via the live log window.
+3. Once scanning the branch has finished, choose the `lane` which will be executed as the main build step by Nevercode.
+4. Finalize the setup by checking the appropriate [build options](https://developer.nevercode.io/docs/build-configuration#section-general-build-settings) and click **Save and start build**.
+## External dependencies
+
+Nevercode does not automatically install CocoaPods, Carthage or any other external dependencies for Fastlane projects. Please use the Fastlane's built-in [actions](https://docs.fastlane.tools/actions/), such as [`cocoapods`](https://docs.fastlane.tools/actions/cocoapods/) and [`carthage`](https://docs.fastlane.tools/actions/carthage/), to gain control of that.
```

Comment:
Does this work? Does fastlane actually install the dependency for the user?

Reply:
Also, `CocoaPods` instead of `cocoapods`.

Reply:
It presents instructions if using `bundler`:

https://github.com/fastlane/fastlane/blob/0c23d5e64f3c193ac013e9383551149ac6e8d88c/fastlane/lib/fastlane/fastlane_require.rb#L14-L22

and automatically installs the gem if not using `bundler`:

https://github.com/fastlane/fastlane/blob/0c23d5e64f3c193ac013e9383551149ac6e8d88c/fastlane/lib/fastlane/fastlane_require.rb#L25-L37